aleph-client/src/contract/mod.rs

```diff
@@ -59,23 +59,42 @@ use crate::{
     AccountId, Balance, ConnectionApi, SignedConnectionApi, TxStatus,
 };
 
+/// Default gas limit, which allows up to 25% of block execution time.
+pub const DEFAULT_MAX_GAS: u64 = 250_000_000_000u64;
+/// Default proof size limit, which allows up to 25% of block execution time.
+pub const DEFAULT_MAX_PROOF_SIZE: u64 = 250_000_000_000u64;
+
 /// Represents a contract instantiated on the chain.
 pub struct ContractInstance {
     address: AccountId,
     transcoder: ContractMessageTranscoder,
+    max_gas_override: Option<u64>,
```

Comment:
Does not really matter, but why not just `max_gas: u64`?

Reply:
so that you can easily back off to the default, by setting to None (it doesn't require knowing `DEFAULT_MAX_GAS`)